app/views/content_items/_content_item.html.erb

```diff
@@ -0,0 +1,4 @@
+<tr>
+  <td><%= link_to content_item.title, content_item.url, target: '_blank' %></td>
```

Comment:
Opening links in a new window is a usability issue (and as I've just learned also a [security issue](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)). If we're going to use it (and I don't see the need here), then it should be made clear (e.g. with a title "opens in new window").